eis_toolkit/conversions/csv_to_geopandas.py

```diff
@@ -0,0 +1,98 @@
+from typing import List
+
+import csv as reader
+import pandas as pd
+import geopandas
+
+from eis_toolkit.exceptions import (InvalidParameterValueException,
+                                    InvalidWktFormatException,
+                                    InvalidColumnIndexException)
+
+
+def _csv_to_geopandas(  # type: ignore[no-any-unimported]
+    csv: str,
+    indexes: List[int],
+    target_EPSG: int,
+) -> geopandas.GeoDataFrame:
+
+    with open(csv, 'r') as f:
```

Comment:
If `pathlib.Path` is used, it exposes an alternative interface for opening a text file:

```suggestion
    with csv.open(mode="r") as f:
```

although the original method should work as well. Just an alternative..